pandas/tests/apply/test_frame_apply.py

```diff
@@ -1585,3 +1585,21 @@ def test_apply_on_empty_dataframe():
     result = df.head(0).apply(lambda x: max(x["a"], x["b"]), axis=1)
     expected = Series([])
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "test, constant",
+    [
+        ({"a": [1, 2, 3], "b": [1, 1, 1]}, {"a": [1, 2, 3], "b": 1}),
+        ({"a": [2, 2, 2], "b": [1, 1, 1]}, {"a": 2, "b": 1}),
```

Comment:
I think the scalars in constant should be contained in lists e.g.

```
# result of agg
First aggregation: <class 'pandas.core.series.Series'>
a    [1, 2, 3]
b          [1]
dtype: object
Second aggregation: <class 'pandas.core.series.Series'>
a    [2]
b    [1]
dtype: object

In [3]: Series(data={"a": [1, 2, 3], "b": 1}, index=["a", "b"], dtype="object")
Out[3]:
a    2
b    1
dtype: object

In [4]: constant = {"a": [1, 2, 3], "b": 1}

In [6]: Series(data=constant, index=["a", "b"], dtype="object")
Out[6]:
a    [1, 2, 3]
b            1
dtype: object
```

Reply:
Sure, I can do that.  Either way will work, but I think you are right that it is more accurate for it to be one-element lists rather than scalars.